kafka/source/pkg/reconciler/source/controller.go

```diff
@@ -33,8 +33,8 @@ import (
 	"knative.dev/pkg/resolver"
 
 	kafkaclient "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/client"
-	kafkainformer "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/informers/sources/v1alpha1/kafkasource"
-	"knative.dev/eventing-contrib/kafka/source/pkg/client/injection/reconciler/sources/v1alpha1/kafkasource"
+	kafkainformer "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/informers/sources/v1beta1/kafkasource"
+	"knative.dev/eventing-contrib/kafka/source/pkg/client/injection/reconciler/sources/v1beta1/kafkasource"
 )
 
 func NewController(
```

Comment:
Can't use "knative.dev/eventing/pkg/apis/sources/v1alpha1" yet as it is not released yet.

Reply:
Not sure if that's a blocker for this PR

Reply:
perhaps go w/ "latest" of the dependency ? (aka master branch)

Reply:
I need to learn more about Go modules :D

Reply:
Let me do this later. Nothing unique from that package is used anyway. Only a `Kind` function which is identical in v1alpha1.
When the dependency to eventing is changed to master I can do it.